README.md

```diff
@@ -206,7 +172,66 @@ this will generate a log at, for example, `log/profile_cv_validate_disk_all_endp
 RAILS_ENV=production bundle exec rake cv_druid[bz514sm9647]
 ```
 
-## Development
+## <a name="seeding"/>Seed the catalog
+
+Seeding the catalog presumes an empty or nearly empty database -- otherwise running the seed task will throw `druid NOT expected to exist in catalog but was found` errors for each found object.
+
+Without profiling:
+```sh
+RAILS_ENV=production bundle exec rake seed_catalog
+```
+
+With profiling:
+```sh
+RAILS_ENV=production bundle exec rake seed_catalog[profile]
+```
+this will generate a log at, for example, `log/profile_seed_catalog_for_all_storage_roots2017-11-13T13:57:01-flat.txt`
+
+
+#### Reset the catalog for re-seeding
+
+WARNING! this will erase the catalog, and thus require re-seeding from scratch.  It is mostly intended for development purposes, and it is unlikely that you'll need to run this against production once the catalog is in regular use.
+
+* Deploy the branch of the code with which you wish to seed, to the instance which you wish to seed (e.g. master to stage).
+* Reset the database for that instance.  E.g., on production or stage:  `RAILS_ENV=production bundle exec rake db:reset`
+  * note that if you do this in an env that sees itself as production (i.e. production or stage), you'll get a scary warning along the lines of:
+  ```
+  ActiveRecord::ProtectedEnvironmentError: You are attempting to run a destructive action against your 'production' database.
+  If you are sure you want to continue, run the same command with the environment variable:
+  DISABLE_DATABASE_ENVIRONMENT_CHECK=1
+  ```
+  Basically an especially inconvenient confirmation dialogue.  For safety's sake, the full command that skips that warning can be constructed by the user as needed, so as to prevent unintentional copy/paste dismissal when the user might be administering multiple deployment environments simultaneously.  Inadvertent database wipes are no fun.
+  * `db:reset` will make sure db is migrated and seeded.  If you want to be extra sure: `RAILS_ENV=[environment] bundle exec rake db:migrate db:seed`
+
+### run `rake db:seed` in a deploy environment:
+
+```sh
+bundle exec cap stage db_seed # for the stage servers
+```
+
+or
+
+```sh
+bundle exec cap prod db_seed # for the prod servers
+```
+
+### Drop or Populate the catalog for a single endpoint
+
+To run either of the rake tasks below, give the name of the moab storage_root (e.g. from settings/development.yml) as an argument.
+
+#### Drop all database entries:
+
+```sh
+RAILS_ENV=production bundle exec rake drop[fixture_sr1]
+```
+
+#### Populate the catalog:
+
+```sh
+RAILS_ENV=production bundle exec rake populate[fixture_sr1]
+```
+
+## <a name="development"/>Development
```

Comment:
Headings automatically have anchors added, so when the anchor isn't changing, it is redundant to add explicitly.  Github already renders the markdown with an anchor, e.g. for `### Redis`, like:
```html
<h3><a id="user-content-redis" class="anchor" aria-hidden="true" href="#redis"></a>Redis</h3>
```

Reply:
ah - I wasn't sure, and couldn't think of a good way to test without pushing to github.

Reply:
merged, filed #832 for this loose end.